go.mod

```diff
@@ -14,43 +14,77 @@
 
 module github.com/sethvargo/terraform-provider-berglas
 
-go 1.16
+go 1.19
 
 require (
-	cloud.google.com/go v0.93.3 // indirect
-	cloud.google.com/go/kms v0.1.0 // indirect
-	cloud.google.com/go/secretmanager v0.1.0 // indirect
-	cloud.google.com/go/storage v1.16.0 // indirect
-	github.com/GoogleCloudPlatform/berglas v0.6.0
+	github.com/GoogleCloudPlatform/berglas v0.6.3-0.20220726215926-83e4b9214bea
```

Comment:
I could not build this provider by using current latest tagged version of berglas client due to following error:

> ../../go/pkg/mod/github.com/!google!cloud!platform/berglas@v0.6.1/pkg/berglas/iam_storage.go:201:12: assignment mismatch: 2 variables but retry.NewFibonacci returns 1 value

This error comes from [this](https://github.com/GoogleCloudPlatform/berglas/blob/main/pkg/berglas/iam_storage.go#L199-L201) change, so I currently refer latest commit of the client.